### DISCUSSION TITLE
networkx 1.11 (node_graph_link) vs networkx 3.1 (node_graph_link)

### DISCUSSION THREAD

**Discussion by ashiqimranintel** [networkx/networkx#6657] [Q&A]
I was using reddit dataset (http://snap.stanford.edu/graphsage/reddit.zip) to do graphsage training on the latest tensorflow. The data pre-processing is working fine with network 1.11 but with latest networkx is behaving weirdly. 

here is the code 
from networkx.readwrite import json_graph
G = json_graph.node_link_graph(G_data,directed=False,multigraph=False)

**Comment by dschult** [comment]
What's the error?

**Comment by rossbar** [comment]
@ashiqimranintel there's not enough info here to dig in and try to figure out what the problems may be. Therefore, I'm going to close this as "can't reproduce". If you can provide more information about the error you are seeing and how to reproduce it, please open a new issue (rather than a discussion).

Also note: networkx 1.11 is >13 years old (!) and has not been supported for nearly as long.
